INSTALL_CIOOS_CKAN.md

```diff
@@ -815,3 +833,36 @@ Copy to volume
 cd ~/ckan/contrib/docker
 sudo cp -r src/ckanext-cioos_theme/ $VOL_CKAN_HOME/venv/src/
 ```
+
+
+### Export Logs from CKAN
+Exporting logs from CKAN allows fail2ban to monitor CKAN and prevent bruteforce attacks.
+
+Edit .env file and update CKAN_LOG_PATH. Path must not have a trailing slash. The path will point to a directory which will be mounted into the CKAN container. Here we assume the path is `/var/log/ckan`
+```bash
+cd ~/ckan/contrib/docker
+nano .env
+```
+
+Change folder permissions on ckan log folder so ckan can write to it.
+sudo mkdir /var/log/ckan
+sudo chmod 770  /var/log/ckan
+sudo chown root:900 /var/log/ckan
+
+If updating an exsiting ckan instance you will need to copy the new entrypoint file into the ckan container. We set owner and permissions using tar stream
+```bash
+tar -cf - ckan-entrypoint.sh --mode u=rwx,g=rx,o=rx --owner root --group root | sudo docker cp - ckan:/
+```
+
+Then restart CKAN
+```bash
+sudo docker-compose restart ckan
+```
+
+If ckan does not start becouse of failed permissions you can reset the container by forcing it to recreate.
+```bash
+sudo docker-compose up -d --force-recreate ckan
+```
+
+### Setup fail2ban on host
+Please contact Matt or Jacob for instructions 
```

Comment:
Are there any existing fail2ban docs out there that we could just link here?

Reply:
yes, and done.